modules/test/playwright/tests/roles-admin-web/organizationRoles.spec.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {dataApiHelpersTest} from '../../fixtures/dataApiHelpersTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {rolesPagesTest} from '../../fixtures/rolesPagesTest';
+import {usersAndOrganizationsPagesTest} from '../../fixtures/usersAndOrganizationsPagesTest';
+import {getRandomInt} from '../../utils/getRandomInt';
+import getRandomString from '../../utils/getRandomString';
+
+export const test = mergeTests(
+	dataApiHelpersTest,
+	loginTest(),
+	rolesPagesTest,
+	usersAndOrganizationsPagesTest
+);
+
+test('LPD-50296 Select Organization Role should show country name, not key', async ({
```

Comment:
can you please use the new mode to tag the ticket numbers? 
```
tags: ['@LPD-50296']
```

Reply:
@smottal , yes I can take care of this.